include/picongpu/particles/ionization/byField/JIonizationCalc.hpp

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2020 Jakob Trojok
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "picongpu/simulation_defines.hpp"
+
+namespace picongpu
+{
+namespace particles
+{
+namespace ionization
+{
+    struct JIonizationCalc
+    {
+        HDINLINE float3_X
+        operator()( float_X const ionizationEnergy, float3_X const eField )
+        {
+            return ionizationEnergy * eField / math::abs(eField) / math::abs(eField) / DELTA_T;
```

Comment:
@jtrojok Are you sure abozt this equation, from a unit comparisom this is:
```
energy * Efield / Efiled^2 / time = energy * (Volt/meter)^-1 * time^-1
= (mass * length^2 * time^-2) * (mass * length * time^-3 * Ampere^-1)^-1 time^-1
= Ampere * meter
!= Ampere 
```

Reply:
I am still confused. According to [1] eq. 21 the implementation is correct - but it should give a current **density** - not a current. However, the units for a current density would be `Ampere * meter^-2`. Do you have to divide by the cell volume?

Reply:
Great you looked at it. There seems to be a division by cell volume missing!

Reply:
changed the formula and tested it. Appears to work